cmd/ftl/cmd_dev.go

```diff
@@ -217,29 +224,27 @@ func (d *devCmd) updateFileInfo(ctx context.Context, dir string, modules moduleM
 			}
 
 			if match && !entry.IsDir() {
-				fileInfo, err := entry.Info()
+				file, err := os.Open(srcPath)
 				if err != nil {
 					return err
 				}
 
-				module := modules[dir]
-				module.NumFiles++
-				if fileInfo.ModTime().After(module.LastModTime) {
-					changed = srcPath
-					module.LastModTime = fileInfo.ModTime()
+				if _, err := io.Copy(fileHash, file); err != nil {
+					file.Close()
```

Comment:
Put `_ = file.Close()` here to show that we're deliberately ignoring this error.